nexus/src/db/datastore.rs

```diff
@@ -2154,6 +2155,120 @@ impl DataStore {
         info!(opctx.log, "created {} built-in users", count);
         Ok(())
     }
+
+    /// List built-in roles
+    pub async fn roles_builtin_list_by_name(
+        &self,
+        opctx: &OpContext,
+        pagparams: &DataPageParams<'_, (String, String)>,
+    ) -> ListResultVec<RoleBuiltin> {
+        use db::schema::role_builtin::dsl;
+        opctx.authorize(authz::Action::ListChildren, authz::FLEET)?;
+
+        // TODO-column This is essentially a multi-column version of
```

Comment:
Re: this comment: I spent a while trying to build a multi-column version of `paginated` but could not get the Diesel trait bounds right.  I gave up on that because there would only be this one caller right now.

Reply:
I have a working version of `paginated_multicolumn` that integrates here. Would you like the commits to be added to this PR, or uploaded as a separate PR?

Reply:
Nice!  I'd love a separate PR just so I can better keep things straight, particularly with #520 also in flight.  Thanks for doing that!

Reply:
(separate PR is #522)